lib/config/typescript-prettier.js

```diff
@@ -4,12 +4,5 @@ module.exports = {
   rules: {
     // rules to disable to prefer prettier
     'typescript/member-delimiter-style': 'off',
```

Comment:
It looks like prettier changed to use ; as member delimiters, to match the eslint rule way back in v 1.4.1 as per https://github.com/prettier/prettier/pull/1918, so I think we can remove this too. But that's can be done in a different PR.

Reply:
makes sense. lets shuffle that in a different PR to keep this one focused.